job-server/src/main/scala/spark/jobserver/WebApi.scala

```diff
@@ -342,6 +343,22 @@ class WebApi(system: ActorSystem,
 
     // user authentication
     authenticate(authenticator) { authInfo =>
+      (get & path(Segment)) { (contextName) =>
+        respondWithMediaType(MediaTypes.`application/json`) { ctx =>
+          val future = supervisor ? GetSparkWebUI(contextName)
+          future.map {
+            case WebUIForContext(name, Some(url)) => ctx.complete(
+              200,
```

Comment:
why not just put the whole `ctx.complete()` on another line so it can all be together on the same line? Not a big deal